platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,50 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
+
+@interface MGLMapViewConversionTests : XCTestCase
+
+@property (nonatomic) UIView *superView;
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation MGLMapViewConversionTests
+
+- (void)setUp {
+    [super setUp];
+    
+    [MGLAccountManager setAccessToken:@"pk.feedcafedeadbeefbadebede"];
+    self.superView = [[UIView alloc] initWithFrame:UIScreen.mainScreen.bounds];
```

Comment:
This isn’t a UI test, so the test runs headless. Therefore, if there’s even a UIScreen to speak of, its bounds would likely be the empty rect. Specify an explicit bounds, which is important anyways for keeping the test consistent across simulators.